### DISCUSSION TITLE
Switching to org.tensorflow:tensorflow-core-platform:1.0.0

### DISCUSSION THREAD

**Discussion by JeffreyKleiss** [Kotlin/kotlindl#584] [General]
I was wondering if there is any discussion to switching to the new tensorflow/java (https://github.com/tensorflow/java) which was just released (1.0.0) and runs Tensorflow 2.16.

Are there any gotchas before trying to start a new branch and updating?

**Comment by RaphaelTarita** [comment]
See https://github.com/Kotlin/kotlindl/issues/537

Because of this, as it currently stands, KotlinDL is essentially dead, sadly

**Reply by dkrieg** [reply]
"KotlinDL is essentially dead, sadly".  Is that so?  What is the precise reason: technical or motivation?  Is it simply not worth pursuing or does this need more community support to transition the solution?
